cmd/mattermost/commands/channel.go

```diff
@@ -533,3 +546,62 @@ func renameChannelCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchChannelCmdF(command *cobra.Command, args []string) error {
+
+	var (
```

Comment:
We don't define the vars at the top of the function, we normally define them, just before the first usage.